code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -771,17 +771,17 @@ \subsubsection{Data Definitions}
               \item{$GTF$ is the glass type factor (Unitless)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & $q$ is the 3 second equivalent pressure, as given in \hyperref[DD:calofDemand]{DD: calofDemand}.
+Notes & The glass is considered to be a single lite (from \hyperref[assumpGL]{A: glassLite}).
```

Comment:
The relevance of this statement is not clear.  In the case study (manual) version, no reference is made to this assumption.  Where did it come from here?  I'm not saying it is wrong, maybe the derivation of qhat depends on this assumption?

Reply:
It was added in #932. Looking at it in more detail now, I don't see the relevance of it either, and the DD in manual doesn't have a derivation that might use this. Should we remove it? I don't think that the change in #932 is justified in adding this assumption.

Reply:
Yes, please remove the assumption here.  Thank you.

Reply:
Done in 4130d37c1.